Source/WebKit/UIProcess/gtk/AcceleratedBackingStoreDMABuf.cpp

```diff
@@ -180,8 +180,21 @@ AcceleratedBackingStoreDMABuf::BufferDMABuf::BufferDMABuf(uint64_t id, const Web
 {
 }
 
-void AcceleratedBackingStoreDMABuf::BufferDMABuf::didUpdateContents()
-{
+void AcceleratedBackingStoreDMABuf::BufferDMABuf::didUpdateContents(Buffer* previousBuffer, const Vector<WebCore::IntRect>& damageRects)
+{
+    if (!damageRects.isEmpty() && previousBuffer && previousBuffer->texture()) {
+        gdk_dmabuf_texture_builder_set_update_texture(m_builder.get(), previousBuffer->texture());
+        RefPtr<cairo_region_t> cairoRegion = adoptRef(cairo_region_create());
+        for (const auto& rect : damageRects) {
+            cairo_rectangle_int_t cairoRect = rect;
```

Comment:
This can be `const`:

```suggestion
            const cairo_rectangle_int_t cairoRect = rect;
```

(It took me a bit to realize that there is an implicit cast operator defined from `IntRect` to `cairo_rectangle_t`, these days I tend to prefer explicit conversions 😉)